lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/02_select_miner_fee.dart

```diff
@@ -106,7 +109,7 @@ class SelectMinerFeeStepState extends State<SelectMinerFeeStep>
 
   void _setSavedFeeData() {
     if (widget.savedFeeType != null) {
-      _setFeeType(widget.savedFeeType?.name);
+      _setFeeType(widget.savedFeeType);
```

Comment:
This is not correctly being set, the fee is not saved when going back from review step.